README.md

```diff
@@ -78,6 +78,9 @@ prediction-dataset/
 cd l5kit
 pip install -r requirements.txt
 ```
+If you run into trouble installing L5Kit on Windows, you may need to
+- install Pytorch and torchvision manually first (select the correct version required by your system, i.e. GPU or CPU-only) then run L5Kit install (remove the packages 'torch' and 'torchvision' from ```setup.py```)
```

Comment:
should we add the links to the 2 lines in the setup.py?

Reply:
good call!